H4LCppModule.py

```diff
@@ -199,12 +199,15 @@ def analyze(self, event):
         passedFiducialSelection=False
         nZXCRFailedLeptons=0
         isMC = True
-        passedTrig = PassTrig(event, 2017)
+        passedTrig = PassTrig(event, 2018)
```

Comment:
don't hard code this here. 

Try to pass the year from here: https://github.com/ram1123/nanoAOD_skim/blob/7b26df21bb996afee7b47d960462e894eff94f9a/post_proc.py